gulpfile.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-console */
+/* eslint-env node */
+
+'use strict';
+
+const { exec } = require('child_process');
+
+const { parallel, watch } = require('gulp');
+
+function build(cb) {
+  exec('make build', (error, stdout, stderr) => {
+    console.error(stderr);
+    console.log(stdout);
+    cb(error);
+  });
+}
```

Comment:
A related issue here is that in development it is common to run `make build` with different values for `SETTINGS_FILE`. It would be useful to have a documented way to run `make dev` with different values for this argument. One option I can suggest is that `make dev SETTINGS_FILE=<value>` would forward the argument to gulp that would then forward it to `make build`.